src/main/java/core/basesyntax/service/ReportServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service;
+
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String TITLE_FOR_RESULT = "fruit,quantity";
+    private static final String NEW_LINE = System.lineSeparator();
+    private static final String PUNCTUATION_MARK = ",";
+
+    @Override
+    public String getReportData(Map<String, Integer> reportMap) {
```

Comment:
You can get data from the storage directly
```suggestion
    public String generateReport() {
```